pandas/core/categorical.py

```diff
@@ -248,7 +248,8 @@ class Categorical(PandasObject):
     __array_priority__ = 1000
     _typ = 'categorical'
 
-    def __init__(self, values, categories=None, ordered=False, fastpath=False):
+    def __init__(self, values, categories=None, ordered=False, fastpath=False,
```

Comment:
this fix should not be here.

instead in core/internals.py/Block/_astype.py

``raise_one_error`` can be a named argument in the signature and not passed to the constructor